runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -391,8 +392,16 @@ private void drawTimerOverlay(Graphics2D graphics, int textX, int textY, GroundI
 			{
 				return;
 			}
+			// Items in the KBD instance use the standard despawn timer
+			else if (isInKBD())
+			{
+				despawnTime = spawnTime.plus(DESPAWN_TIME_LOOT);
```

Comment:
We need to check `LootType.DROPPED` to determine whether the despawn time should be set to `DESPAWN_TIME_LOOT` (2 minutes) or `DESPAWN_TIME_DROP` (3 minutes).

Reply:
Thanks for the catch, for some reason I was thinking that the timers would be the same like they would be in an instance just shorter.